src/Robo/Hooks/ValidateHook.php

```diff
@@ -131,4 +131,22 @@ public function validateVmConfig() {
     }
   }
 
+  /**
+   * Validates that Git user is configured.
+   *
+   * @hook validate @validateGitConfig
+   */
+  public function validateGitConfig() {
+    if (!$this->getInspector()->isGitMinimumVersionSatisfied('2.0')) {
+      throw new BltException("Your system does not meet BLT's requirements. Please update git to 2.0 or newer.");
+    }
+    if (!$this->getInspector()->isGitUserSet()) {
+      if (!$this->getConfigValue('git.user.name') || !$this->getConfigValue('git.user.email')) {
```

Comment:
Can't you prompt the user for these values and set defaults in case -n ?

Reply:
Yeah, I thought about that but wasn't sure the best way. Normally we use interact hooks to get user input, but they don't run during `--no-interaction`, so we'd have to duplicate a bit of code between the interact and validate hooks, which seems needlessly complex. It doesn't seem kosher to solicit for user input during a validate hook, but I don't see a better option.